Common.props

```diff
@@ -1,11 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Global.props'))\Global.props" />
-  <Import Project=".\GlobalStaticVersion.props" />
```

Comment:
It is the same problem but alternative solution. I like this one better because it does not require non-default project structure (i.e. you can say `<Project Sdk="Microsoft.NET.Sdk">` and everything still works.

Not sure exactly what you mean "align by names of files"? Do you mean "make it consistent between the two repos"?